### PR TITLE
fix: active_model_serializers installer

### PR DESCRIPTION
**Pull request by robertlaurin**
Small fix for the `active_model_serializers` instrumentation.

### CODE REVIEW

File: instrumentation/active_model_serializers/lib/opentelemetry/instrumentation/active_model_serializers/instrumentation.rb [hunk 1]
```diff
@@ -19,13 +19,14 @@ class Instrumentation < OpenTelemetry::Instrumentation::Base
         present do
           !defined?(::ActiveSupport::Notifications).nil? &&
             !defined?(::ActiveModel::Serializers).nil? &&
```
Comment (robertlaurin) [inline instrumentation/active_model_serializers/lib/opentelemetry/instrumentation/active_model_serializers/instrumentation.rb:21]: ActiveModel actually defines this namespace without the presence of the `active_model_serializers` gem 

https://github.com/rails/rails/blob/master/activemodel/lib/active_model.rb#L62

File: instrumentation/active_model_serializers/lib/opentelemetry/instrumentation/active_model_serializers/instrumentation.rb [hunk 2]
```diff
@@ -19,13 +19,14 @@ class Instrumentation < OpenTelemetry::Instrumentation::Base
...
+            !defined?(::ActiveModelSerializers).nil? &&
```
Comment (robertlaurin) [inline instrumentation/active_model_serializers/lib/opentelemetry/instrumentation/active_model_serializers/instrumentation.rb:22]: This is the error that comes up on a rails app without `active_model_serializers`
```
WARN -- : Instrumentation: OpenTelemetry::Instrumentation::ActiveModelSerializers unhandled exception during install undefined method `>=' for nil:NilClass
```

Comment (robertlaurin) [reply instrumentation/active_model_serializers/lib/opentelemetry/instrumentation/active_model_serializers/instrumentation.rb:22]: Hey @johnnyshields does this check look right?  I haven't spent any time with this gem before.

Comment (johnnyshields) [reply instrumentation/active_model_serializers/lib/opentelemetry/instrumentation/active_model_serializers/instrumentation.rb:22]: @robertlaurin yep this change is correct

Comment (johnnyshields) [reply instrumentation/active_model_serializers/lib/opentelemetry/instrumentation/active_model_serializers/instrumentation.rb:22]: you can actually remove `!defined?(::ActiveModel::Serializers).nil?` if you want, it's redundant.

Comment (robertlaurin) [reply instrumentation/active_model_serializers/lib/opentelemetry/instrumentation/active_model_serializers/instrumentation.rb:22]: Will do, thanks for looking it over.

**Review by fbogsany — APPROVED** [review]

**Pull request merged**